examples/js/loaders/GLTFLoader.js

```diff
@@ -305,7 +317,8 @@ THREE.GLTFLoader = ( function () {
 		KHR_MATERIALS_PBR_SPECULAR_GLOSSINESS: 'KHR_materials_pbrSpecularGlossiness',
 		KHR_MATERIALS_UNLIT: 'KHR_materials_unlit',
 		KHR_TEXTURE_TRANSFORM: 'KHR_texture_transform',
-		MSFT_TEXTURE_DDS: 'MSFT_texture_dds'
+		MSFT_TEXTURE_DDS: 'MSFT_texture_dds',
+		MOZ_HUBS_TEXTURE_BASIS: "MOZ_HUBS_texture_basis"
```

Comment:
This should be `MOZ_hubs_texture_basis` we do not have the `MOZ_HUBS` vendor prefix registered.